domains/environment_domains/variables.tf

```diff
@@ -2,6 +2,7 @@ variable "zone" {}
 variable "front_door_name" {}
 variable "resource_group_name" {}
 variable "domains" {}
+variable "exclude_domains_cname" { default = [] }
```

Comment:
Would it be clearer to name it `exclude_cname_records` or `exclude_cnames` or... ?

Reply:
Can we add a description? It a good practice we should start following

Reply:
Added and updated